temp.py

```diff
@@ -0,0 +1,8 @@
+def reset_files(filepath, file_type="txt"):
+    with open(filepath, "w") as output_file:
+        if file_type == "txt":
+            output_file.write("")
+        elif file_type == "json":
+            output_file.write(json.dumps({}))
+        else:
+            log.error("ERROR: Invalid file type")
```

Comment:
Consider using a more robust logging mechanism instead of `log.error` which may not be defined. Ensure that the logging is set up correctly or use the `logging` module. [important]